bears/python/DocformatterBear.py

```diff
@@ -0,0 +1,60 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='docformatter',
+        output_format='unified-diff',
+        diff_distance=-1,
+        prerequisite_check_command=('docformatter', '--version'),
+        prerequisite_check_fail_message='docformatter is not installed. Run '
+                                        '`pip install docformatter` to install'
+                                        'docformatter.',
+        result_message='The documentation string does not meet PEP 257 '
+                       'conventions.')
+class DocformatterBear:
+    """
+    Check the docstrings according to PEP 257 conventions.
+
+    More information available at https://github.com/myint/docformatter
+    """
+
+    LANGUAGES = {'Python', 'Python 2', 'Python 3'}
+    REQUIREMENTS = {PipRequirement('docformatter')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting', 'Documentation'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         wrap_summaries_length: int=79,
+                         wrap_descriptions_length: int=72,
```

Comment:
if 72 is relative to 79, then this default should be `None`, and then it can be calculated by default as relative to whatever the value of setting `wrap_summaries_length` is chosen by the user.